com.ca.lsp.cobol/lsp-core-domain/src/test/java/com/broadcom/lsp/domain/CopybookStorableProvider.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ *
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ * Broadcom, Inc. - initial API and implementation
+ *
+ */
+package com.broadcom.lsp.domain;
+
+import com.broadcom.lsp.domain.cobol.databus.model.CopybookStorable;
+import com.broadcom.lsp.domain.cobol.event.api.CopybookObserver;
+import com.broadcom.lsp.domain.cobol.event.model.DataEvent;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.inject.Singleton;
+
+@Slf4j
+public class CopybookStorableProvider implements CopybookObserver<DataEvent> {
+  @Getter private static final String unknownEventMessage = "Dummy event";
```

Comment:
Please, make these names all capitals due to the naming convention. Also, do we need getters for them?